.circleci/config.yml

```diff
@@ -609,6 +609,47 @@ jobs:
       - <<: *STEP_ATTACH_WORKSPACE
       - run: sh dockerfiles/verify_packages/verify_<< parameters.package_type >>.sh
 
+  pecl_build:
+    working_directory: ~/datadog
+    executor:
+      name: with_agent
+      docker_image: "datadog/dd-trace-ci:php-7.4-debug-buster"
+    steps:
+      - <<: *STEP_ATTACH_WORKSPACE
+      - <<: *STEP_COMPOSER_INSTALL
```

Comment:
We need this to generate the `bridge/_generated.php` file, right? Kind of bothers me that we need _composer_ to build the _pecl package_ but at least users of pecl don't need it.

Reply:
Correct on both accounts.